nginx.conf

```diff
@@ -0,0 +1,90 @@
+#user  nobody;
+#Defines which Linux system user will own and run the Nginx server
+
+worker_processes  1;
+#Referes to single threaded process. Generally set to be equal to the number of CPUs or cores.
+
+#error_log  logs/error.log; #error_log  logs/error.log  notice;
+#Specifies the file where server logs. 
+
+#pid        logs/nginx.pid;
+#nginx will write its master process ID(PID).
+
+events {
+    worker_connections  1024;
+    # worker_processes and worker_connections allows you to calculate maxclients value: 
+    # max_clients = worker_processes * worker_connections
+}
+
+
+http {
+    include       mime.types;
+    # anything written in /opt/nginx/conf/mime.types is interpreted as if written inside the http { } block
+
+    default_type  application/octet-stream;
+    #
+
+    #log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+    #                  '$status $body_bytes_sent "$http_referer" '
+    #                  '"$http_user_agent" "$http_x_forwarded_for"';
+
+    #access_log  logs/access.log  main;
+
+    sendfile        on;
+    # If serving locally stored static files, sendfile is essential to speed up the server,
+    # But if using as reverse proxy one can deactivate it
+    
+    #tcp_nopush     on;
+    # works opposite to tcp_nodelay. Instead of optimizing delays, it optimizes the amount of data sent at once.
+
+    #keepalive_timeout  0;
+    keepalive_timeout  65;
+    # timeout during which a keep-alive client connection will stay open.
+
+    #gzip  on;
+    # tells the server to use on-the-fly gzip compression.
+
+    server {
+        location / {
+            root /app/floodviz/build/;
```

Comment:
I think it'd be less confusing for future developers to know where the `build` directory is coming from. Folks who look at the requirements and see Frozen-Flask will realize where the build directory is coming from, but it does require some sleuthing and familiarity with the project; it new developer working on the NGINX config sometime to figure all that out.